AssetInformationListener/Infrastructure/DbEntity.cs

```diff
@@ -0,0 +1,25 @@
+using Amazon.DynamoDBv2.DataModel;
+using Hackney.Core.DynamoDb.Converters;
+using System;
+
+namespace AssetInformationListener.Infrastructure
+{
+    // TODO - Alter if DynamoDb is not required
```

Comment:
Remove comment as we are using Dynamodb

Reply:
It will be when the next ticket is done. This is just setting up the application shell.